src/main/java/com/reckue/post/services/realizations/PostServiceRealization.java

```diff
@@ -39,9 +40,30 @@ public Post create(Post post) {
         if (post.getNodes() != null) {
             post.getNodes().forEach(nodeService::create);
         }
+        validateOnCreateStatus(post);
         return postRepository.save(post);
     }
 
+    private void validateOnCreateStatus(Post post) {
+        if (post != null && post.getStatus() == PostStatusType.DRAFT) {
```

Comment:
Don't understand the logic. Does it work?
I think that this method should create a status DRAFT for post, but it check existing status of post. But if the post has just been created, does the post have a status? i think it should be generated automatically.

Reply:
When creating, we will always have a draft status

Reply:
When creating, we will always have a draft status